RxAlamofireTests/RxAlamofireTests.swift

```diff
@@ -109,4 +109,40 @@ class RxAlamofireSpec: XCTestCase {
         let different = RxProgress(bytesWritten: 2000, totalBytes: 4000)
         XCTAssertNotEqual(subject, different)
     }
+    
+    func testDownload() {
+        do {
+            let temporaryDirectory = URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
+            let fileURL = temporaryDirectory.appendingPathComponent("\(UUID().uuidString).json")
+            
+            let destination: DownloadRequest.DownloadFileDestination = { _, _ in (fileURL, []) }
+            let request = download(
+                "http://myjsondata.com",
+                to: destination
+            )
+            
+            let defaultResponse = try request.rx
+                .response()
+                .toBlocking()
+                .first()!
+            
+            XCTAssertEqual(defaultResponse.response?.statusCode, 200)
+            XCTAssertNotNil(defaultResponse.destinationURL)
+            
+            let jsonResponse = try request.rx
```

Comment:
can we split the test to be `testDownloadResponse` & `testDownloadResponseSerialized`?

Reply:
Sure!